domain/src/main/java/org/oppia/domain/topic/TopicListController.kt

```diff
@@ -0,0 +1,118 @@
+package org.oppia.domain.topic
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import org.oppia.app.model.LessonThumbnail
+import org.oppia.app.model.LessonThumbnailGraphic
+import org.oppia.app.model.OngoingStoryList
+import org.oppia.app.model.PromotedStory
+import org.oppia.app.model.TopicList
+import org.oppia.app.model.TopicSummary
+import org.oppia.util.data.AsyncResult
+import java.util.concurrent.TimeUnit
+import javax.inject.Inject
+import javax.inject.Singleton
```

Comment:
I think we are following alphabetical order for imports. Please check this.

Reply:
Done. I think that AS is automatically putting java/javax imports at the bottom of the import list.